src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -0,0 +1,137 @@
+# NTFS to Netex France conversion
+
+Version of the implementation: `1.0`
+
+## Introduction
+
+This document describes how a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) is transformed into a Netex profil France feed in Navitia Transit Model.
+
+The resulting ZIP archive is composed of:
+- a `arrets.xml` file containing the description of all stops (Quays and StopPlaces)
+- a `correspondances.xml` file containing all transfers between stops
+- a folder for each network containing
+  + TBD
+
+## Input parameters
+
+A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
+- `ParticipantRef` : The ParticipantRef code must be provided,
+- `StopProviderCode` : The code used to identify the provider in the stop_id generation.
+
+## File headers
+
+Each Netex file starts with a header containing some information generated by the program: 
+- `/PublicationDelivery/@version`: **x.y:FR-NETEX_nnnn-a.b-c** with
+  + `x.y`:  the version of Netex XSD of Netex (actually 1.09)
```

Comment:
you mean `currently 1.09` ?

Reply:
thanks ! I'm always mistaking on this one !